syntax/tests/printer/other/expected/ocamlString.res.txt

```diff
@@ -9,18 +9,18 @@ let x = "foo\o012bar"
 let x = "😁 this works now 😆"
 let x = `😁 this works now 😆`
 
-/* The `//` should not result into an extra comment */
+// The `//` should not result into an extra comment
 let x = j`https://www.apple.com`
 let x = `https://www.apple.com`
 let x = `https://www.apple.com`
 let x = `https://www.apple.com`
 let x = sql`https://www.apple.com`
 
-/* /* */ should not result in an extra comments */
+// /* */ should not result in an extra comments
 let x = j`/* https://www.apple.com */`
 let x = `/* https://www.apple.com*/`
 let x = `/*https://www.apple.com*/`
 let x = `/*https://www.apple.com*/`
 let x = sql`/*https://www.apple.com*/`
 
-let x = `\`https://\${appleWebsite}\``
```

Comment:
@tsnobip I'm not sure what the ReScript equivalent of
```
{js|`https://${appleWebsite}`|js}
```
should be.  I tried
```
`\`https://${appleWebsite}\``
```
but now the `$` isn't escaped in the output anymore.  TBH, I 'm not really sure what the original test case was trying to test.

Reply:
@kevinbarabash `{js|` is just a unicode string with no string-interpolation, in rescript master, you can just use regular double quotes `"` for that 
```
"https://${appleWebsite}"
```
should do it.

Reply:
The original had backticks between between the `{js|` and `|js}`.  Wouldn't that make it 
```
"`https://${appleWebsite}`"
```
of does `{js|` does something special with backticks?

Reply:
oops sorry my bad, it is indeed 
```res
"`https://${appleWebsite}`"
```